builtin/provisioners/local-exec/resource_provisioner_test.go

```diff
@@ -58,26 +58,35 @@ func TestResourceProvider_stop(t *testing.T) {
 	output := new(terraform.MockUIOutput)
 	p := Provisioner()
 
-	var err error
 	doneCh := make(chan struct{})
+	startTime := time.Now()
 	go func() {
 		defer close(doneCh)
-		err = p.Apply(output, nil, c)
+		// The functionality of p.Apply is tested in TestResourceProvider_Apply.
+		// Because p.Apply is called in a goroutine, trying to t.Fatal() on its
+		// result would be ignored.
```

Comment:
Small nit, the `testing` package would fail with `panic: Fail in goroutine after TestResourceProvider_stop has completed` if you call `t`'s methods after the test completes.